src/styles/main.scss

```diff
@@ -0,0 +1,76 @@
+$day-size: 100px;
+$day-border: 1px;
+$day-color: #eee;
+$day-color-hover: #ffbfcb;
+$day-font: Arial, sans-serif;
+$day-font-size: 30px;
+$gap: 1px;
+$columns: 7;
+$columns-size: $day-size + $day-border * 2;
+$max-width: $columns * $day-size + (($columns - 1) * $gap);
+
+body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
```

Comment:
The body styles are correctly set to center the content vertically and horizontally using flexbox, which aligns with the task requirement to center the calendar on the screen.